en/docs/chapter_searching/binary_search_insertion.md

```diff
@@ -32,21 +32,21 @@ Therefore, at the end of the binary, it is certain that: $i$ points to the first
 
     Based on the previous question, assume the array may contain duplicate elements, all else remains the same.
 
-Suppose there are multiple `target`s in the array, ordinary binary search can only return the index of one of the `target`s, **and it cannot determine how many `target`s are to the left and right of that element**.
+Suppose there are multiple `target`s in the array, a regular binary search can only return the index of one `target`, **and it cannot determine how many `target`s are to the left and right of that it**.
```

Comment:
When there are multiple occurrences of `target` in the array, a regular binary search can only return the index of one occurrence of `target`, and it cannot determine how many occurrences of `target` are to the left and right of that position.